src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSemanticText.java

```diff
@@ -84,7 +84,7 @@ synchronized void foo() {
     static {
 
         int a = 0;}
-    // violation below, Empty block
+    // violation below
```

Comment:
Do you see a reason why messages are supported for violation above/below ?

Reply:
Example https://github.com/checkstyle/checkstyle/pull/11051/files#diff-52fde5cee31a03e135e9c918bdde51ab828aa8d2a4fd8cc9a74f9f651985961eR42

Reply:
Yeah I will specify the violation msg for the above/below